google/ggrc-core
# GGRC-6551 "Cycle", "Cycle Task Group", "Cycle Task" are missed in the "Mapped to" list on Export page

# Issue description

"Cycle", "Cycle Task Group", "Cycle Task" are missed in the "Mapped to" list on Export page

# Steps to test the changes

Steps to reproduce:

1. Open Export page.
2. Expand "Mapped to" list.
 

Actual result: Cycle, Cycle Task Group, Cycle Task are missed in the list.
Expected result: Cycle, Cycle Task Group, Cycle Task should be presented in the list. Search must work for all types by the following type fields: `title`, `name`, `email`, `notes`, `description`, `slug` (if corresponding fields exist)

# Solution description

Cycle, Cycle Task Group, Cycle Task were added to the "Mapped to" list.

# Sanity checklist




# PR Review checklist
